src/components/NcListItem/NcListItem.vue

```diff
@@ -394,21 +394,20 @@
 							</div>
 						</div>
 					</div>
+					<!-- Actions -->
```

Comment:
This **must not** be done! An interactive element (the `a`) must **not** contain any other interactive element (the `NcActions`).
We changed this just a couple of weeks ago.